scylla-cql/src/frame/value.rs

```diff
@@ -64,7 +64,10 @@ pub enum MaybeUnset<V> {
 /// Bytes provided by the user via constructor are passed to DB as is.
 /// As of now, users are guaranteed to fetch the exact same data they inserted
 /// previously (even if it was not normalized).
-#[derive(Clone, Debug)]
+///
+/// The implementation of [`PartialEq`], however, normalizes the underlying bytes
+/// before comparison. For details, check [examples](#impl-PartialEq-for-CqlVarint).
+#[derive(Clone, Eq, Debug)]
```

Comment:
Does `num_bigint::BigInt` implement `Hash`? If so, then I think we should implement it, too.